docs/src/main/sphinx/release/release-397.md

```diff
@@ -0,0 +1,46 @@
+# Release 397 (21 Sep 2022)
+
+## General
+
+* Fix incorrect parsing of invalid values in cast from `varchar` to `timestamp`. ({issue}`14164`)
+* Fix potential incorrect results for queries where all data scanned from tables
+  was processed in the leaf stage and was not passed downstream. ({issue}`14168`)
+* Fix `EXPLAIN (TYPE IO)` query failure for certain queries with redundant table
+  scans. ({issue}`10398`)
+
+## JDBC driver
+
+* Fix potential memory leak when cancelling statements. ({issue}`14176`)
+
+## Delta Lake connector
+
+* Add `parquet.optimized-writer.enabled` as the new configuration property for
+  enabling the optimized parquet writer. Deprecate the
+  `parquet.experimental-optimized-writer.enabled` configuration property. ({issue}`14137`)
+
+## Hive connector
+
+* Add `parquet.optimized-writer.enabled` as the new configuration property for
+  enabling the optimized parquet writer. Deprecate the
+  `parquet.experimental-optimized-writer.enabled` configuration property. ({issue}`14137`)
+* Add support for Hive splits for JSON inputs in S3 Select. ({issue}`14040`)
+* Improve planning time for Glue tables with a large number of columns. ({issue}`14206`)
```

Comment:
Glue tables?? You mean tables in S3 or wherever when the metadata is in Glue .. so thats not a Glue table then right?